core/src/main/java/org/geysermc/geyser/registry/populator/BlockRegistryPopulator.java

```diff
@@ -428,6 +561,45 @@ private static void registerJavaBlocks() {
         }
         BlockStateValues.JAVA_WATER_ID = waterRuntimeId;
 
+        if (BlockRegistries.NON_VANILLA_BLOCK_STATE_OVERRIDES.get().size() > 0) {
+            Set<Integer> usedNonVanillaRuntimeIDs = new HashSet<>();
+            Set<Integer> usedUniqueRuntimeIDs = new HashSet<>();
```

Comment:
`usedUniqueRuntimeIDs` seems to be unused.

Reply:
https://github.com/GeyserMC/Geyser/pull/3505/commits/ba7708f25b97b5bc07c31a3be48b3f31cc1df6a0#diff-c0a896602d5d2af0f1332c0ccc71deea24882f7b0c8087f6df9928a7eb13382eL566